massgen/filesystem_manager/_docker_manager.py

```
@@ -1029,6 +1029,157 @@ def log_container_info(self, agent_id: str) -> None:
+            return {
+                "exists": True,
+                "status": container.status,
+                "running": state.get("Running", False),
+                "exit_code": state.get("ExitCode"),
+                "error": state.get("Error") or None,
+                "started_at": state.get("StartedAt"),
+                "finished_at": state.get("FinishedAt"),
+                "oom_killed": state.get("OOMKilled", False),
+                "pid": state.get("Pid"),
+            }
+        except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)